modules/machineset-creating.adoc

```diff
@@ -0,0 +1,261 @@
+----
+
+. In `<file_name>.yaml`, delete the `status` stanza:
++
+[source,yaml]
+----
+status:
+  availableReplicas: 1
+  fullyLabeledReplicas: 1
+  observedGeneration: 1
+  readyReplicas: 1
+  replicas: 1
+----
+
+. In `<file_name>.yaml`, update both instances of the `sigs.k8s.io/cluster-api-machineset` parameter
+values in the `spec` section to match the `name` that you defined in the `metadata` section:
++
+[source,yaml]
+----
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-cluster: cluster_name
+      sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        sigs.k8s.io/cluster-api-cluster: <cluster_name>
+        sigs.k8s.io/cluster-api-machine-role: <machine_label>
+        sigs.k8s.io/cluster-api-machine-type: <machine_label>
+        sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+...
+----
+<1> Ensure that both the `sigs.k8s.io/cluster-api-machineset` parameter values
+match the `name` that you defined in the `metadata` section.
+
+. In `<file_name>.yaml`, add the node label definition to the spec. The label
+definition resembles the following stanza:
++
+[source,yaml]
+----
+  spec:
+    metadata:
+      labels:
+        node-role.kubernetes.io/<label_name>: "" <1>
+----
+<1> In this definition, `<label_name>` is the node label to add. For example, to
+add the `infra` label to the nodes, specify `node-role.kubernetes.io/infra`.
++
+The updated `spec` section resembles this example:
++
+[source,yaml]
+----
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-cluster: cluster_name
+      sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone>
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        sigs.k8s.io/cluster-api-cluster: <cluster_name>
+        sigs.k8s.io/cluster-api-machine-role: <machine_label>
+        sigs.k8s.io/cluster-api-machine-type: <machine_label>
+        sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone>
+    spec: <1>
+      metadata:
+        labels:
+          node-role.kubernetes.io/<label_name>: ""
+...
+----
+<1> Place the `spec` stanza here.
+
+. Optionally, modify the EC2 instance type and modify the storage volumes.
++
+[IMPORTANT]
+====
+Take care to modify only the parameters that describe the EC2 instance type
+and storage volumes. You must not change the other parameters value in the
+`providerSpec` section.
+====
++
+[source,yaml]
+----
+providerSpec:
+  value:
+    ami:
+      id: ami-0e2bcd33dfff9c73e <1>
+    apiVersion: awsproviderconfig.k8s.io/v1alpha1
+    blockDevices: <2>
+    - ebs:
+        iops: 0
+        volumeSize: 120
+        volumeType: gp2
+    deviceIndex: 0
+    iamInstanceProfile: <3>
+      id: <cluster_name>-<machine_label>-profile
```

Comment:
@kalexand-rh It *appears* that you need to leave id: `<machine_label>` in this string as `worker` or `master` and not the new label (unless there is somewhere to register the new label). My new label wasn't recognized here; when I switched to `worker` it seemed to work.